### PR TITLE
fix: example form_token not working in actix-web 4.3.0

### PR DESCRIPTION
**Pull request by yumm007**
1.Enable feature "std" for crate base64 . src/token_rng.rs:29:20 base64::encode_config need this feature. 
2.Change csrf-cookie name to "Csrf-Token", if use default value __Host-Csrf-token, the lasted version browser(Safari, Chrome) will not send this cookies to server.
3. Add csrf cookie_config to actin-web app data, otherwise crate can't find csrf token.

### CODE REVIEW

**Comment by edward-shen** [comment]
Hi! Thank you for the contribution.

> Enable feature "std" for crate base64 .

Thank you for catching this!

> Change csrf-cookie name to "Csrf-Token", if use default value __Host-Csrf-token, the lasted version browser(Safari, Chrome) will not send this cookies to server.

How are you verifying this? This example needs to be served over HTTPS, not HTTP. If you just do `cargo run --example form_token`, it will be served over HTTP.

**Comment by yumm007** [comment]
Thank you!

I has check it and it work on https, so I push a new commit change example form_token to https.

File: Cargo.toml [hunk 1]
```diff
@@ -12,18 +12,20 @@ include = ["src/**/*", "LICENSE-*", "README.md"]
...
 
 [dev-dependencies]
 anyhow = "1"
-actix-web = { version = "4", features = [ "cookies" ] }
+actix-web = { version = "4", features = [ "cookies", "openssl"] }
 actix-http = "3"
 serde = { version = "1", features = [ "derive" ] }
 tokio = { version = "1", features = [ "macros", "rt-multi-thread" ] }
+openssl = { version = "0.10", features = ["vendored"] }
+
```
Comment (edward-shen) [inline Cargo.toml:28]: Remove newline please!

```suggestion
```

File: examples/form_token.rs [hunk 2]
```diff
@@ -9,10 +9,29 @@ use actix_web::http::Method;
 use actix_web::web::Form;
 use actix_web::HttpResponse;
 use actix_web::{get, post, App, HttpServer, Responder};
+use openssl::{
+    pkey::PKey,
+    ssl::{SslAcceptor, SslAcceptorBuilder, SslMethod},
+    x509::X509,
+};
```
Comment (edward-shen) [inline examples/form_token.rs]: Sorry, can you match the existing import style?

```suggestion
use openssl::pkey::PKey;
use openssl::ssl::{SslAcceptor, SslAcceptorBuilder, SslMethod};
use openssl::x509::X509;
```

**Review by edward-shen — APPROVED** [review]
lgtm, please address final nits and I'll merge!

**Comment by yumm007** [comment]
Thank you! I has change the import style.

**Comment by edward-shen** [comment]
Thanks again for the contribution!

**Pull request merged**